README.md

```diff
@@ -44,6 +44,9 @@ run style checks use `composer cs`.
 
 ## Release notes
 
+### Next Version (2.0.x)
+* Ability ro re-ruse the file `cli-config.php` when including `FundraisingStore` in an application is removed.
```

Comment:
I have no strong preference either way. Since we're the only users, and nothing will break if we release without bumping the version to 3.x (right?), we could just decide to not care about semver in this case.